audit/syslogtls.go

```diff
@@ -32,16 +33,7 @@ type SyslogParams struct {
 // NewSyslogTLSTarget creates a target capable of outputting log records to remote or local syslog via TLS.
 func NewSyslogTLSTarget(filter logr.Filter, formatter logr.Formatter, params *SyslogParams, maxQueue int) (*SyslogTLS, error) {
 	config := tls.Config{InsecureSkipVerify: params.Insecure}
-
-	if params.Cert != "" {
-		serverCert, err := ioutil.ReadFile(params.Cert)
-		if err != nil {
-			return nil, err
-		}
-		pool := x509.NewCertPool()
-		pool.AppendCertsFromPEM(serverCert)
-		config.RootCAs = pool
-	}
+	config.RootCAs = getCertPool(params.Cert)
```

Comment:
I don't have context into who is using this, but is it feasible to distinguish a base-64 encoded certificate from a file as a separate params property instead of trying to test both possibilities?

Reply:
Thanks for reviewing @lieut-data .  This is how cloud team asked for it, and as I recall is in line with other fields they use the same way.  @corey-robinson any thoughts on this?

On thing to consider is there would be extra logic needed for separate fields, such as erroring if both are used.

Reply:
Ah, that makes sense, and seems reasonable. Thanks for the clarity, @wiggin77!